generators/app/constants.js

```diff
@@ -63,10 +63,14 @@ module.exports.FLOWCONFIG_PATH = {
   destination: '.flowconfig'
 };
 
+module.exports.RESCRIPTS_PATH = {
```

Comment:
I think that we can found a better approach to avoid to create a new object per new template config file